libs/components/forms/src/lib/modules/input-box/input-box-host.service.ts

```diff
@@ -60,4 +60,14 @@ export class SkyInputBoxHostService {
 
     this.#host.setHintTextHidden(hide);
   }
+
+  public setHintTextRemoved(hide: boolean): void {
```

Comment:
Same question that I posed on the input box component regarding the mixing of the `ScreenReaderOnly` and `Removed` terminology.

Reply:
changed